src/Pagination.js

```diff
@@ -0,0 +1,240 @@
+'use strict';
+
+import core from 'metal';
+import dom from 'metal-dom';
+import templates from './Pagination.soy';
+import Component from 'metal-component';
+import Soy from 'metal-soy';
+
+class Pagination extends Component  {
+	constructor(value) {
+		super(value);
+
+		this.TOTAL_CONTROLS = 2;
```

Comment:
I can't seem to find anywhere that's using this. If it really isn't supposed to be used, it's best to remove.